packages/effects-docs/src/pages/examples.js

```diff
@@ -38,7 +38,7 @@ const Examples = () => {
               <td>{i + 1}</td>
               <td>
                 <Link
-                  to={`examples/${withoutExampleSuffix(
+                  to={`/examples/${withoutExampleSuffix(
```

Comment:
AND THIS lack of `/` was what was causing the warning message from gatsby's dev runtime.